internal/nodes/ingress_model_rel.go

```diff
@@ -578,11 +583,45 @@ func HTTPRuleToIng(rrname string, namespace string, key string) ([]string, bool)
 		}
 	}
 
-	utils.AviLog.Infof("key: %s, msg: ingresses to compute: %v via httprule %s",
-		key, allIngresses, namespace+"/"+rrname)
+	utils.AviLog.Debugf("key: %s, msg: Ingresses retrieved %s", key, allIngresses)
 	return allIngresses, true
 }
 
+func NsxAlbSettingToGateway(albSettingName string, namespace string, key string) ([]string, bool) {
+	allGateways := make([]string, 0)
+
+	_, err := lib.GetCRDInformers().NsxAlbInfraSettingInformer.Lister().Get(albSettingName)
+	if err != nil {
```

Comment:
Should we check for a deleted case ?
And why is this call being made if we are not using the returned object ?

Reply:
added an explicit object not found check. Nice catch!